samples/nrf9160/at_cmd_parser/src/main.c

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
```

Comment:
I do not think you need to include `stdio`, `uart` or `string`.

Reply:
Are you sure? It works without these includes right now, but is that guaranteed by zephyr.h?

Reply:
Based on the types you are using, `stddef.h` is needed.